src/output/icons.rs

```diff
@@ -196,9 +200,9 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     ".tcshrc"             => Icons::SHELL,          // 󱆃
     ".viminfo"            => Icons::VIM,            // 
     ".vimrc"              => Icons::VIM,            // 
-    ".Xauthority"         => Icons::CONFIG,         // 
-    ".xinitrc"            => Icons::CONFIG,         // 
-    ".Xresources"         => Icons::CONFIG,         // 
+    ".Xauthority"         => Icons::CONFIG,         // 󱁻
```

Comment:
Hi, for the `.Xauthority`, `.xinitrc` and `.Xresources` files I would recommend to use the X.org glyph. The codepoint is `'\u{f369}'`

_Note: With the release of NF 3.1.0 a lot of new icons from `font-logos` project were added._